pipeline.ml

```diff
@@ -12,118 +12,144 @@ let read_fpath p = Bos.OS.File.read p |> Rresult.R.error_msg_to_invalid_arg
 let read_channel_uri p =
   read_fpath p |> String.trim |> Uri.of_string |> Current_slack.channel
 
+let pool = Current.Pool.create ~label:"docker" 1
+
+let get_commit ~head =
+  let { Github.Repo_id.owner; name } = Github.Api.Commit.repo_id head in
+  let hash = Github.Api.Commit.hash head in
+  let res = hash ^ owner ^ name in
+  res
+
+let merge_json bench head =
+  let+ head = head in
+  let commit_string = get_commit ~head in
+  Yojson.Basic.to_string
+    (`Assoc
+      [ ("commit", `String commit_string); ("benchmarks", `String bench) ])
+
 (* Generate a Dockerfile for building all the opam packages in the build context. *)
 let dockerfile ~base =
   let open Dockerfile in
   from (Docker.Image.hash base)
   @@ run
        "sudo apt-get install -qq -yy libffi-dev liblmdb-dev m4 pkg-config \
-        gnuplot-x11"
+        gnuplot-x11 capnproto libcapnp-dev libgmp-dev libsqlite3-dev"
   @@ copy ~src:[ "--chown=opam:opam ." ] ~dst:"index" ()
   @@ workdir "index"
+  @@ run "ls"
```

Comment:
Are these `ls`s in here for debugging purposes?

Reply:
yarp, will remove!